### PR TITLE
Document training instructions

### PR DESCRIPTION
**Pull request by Thomsch**
This PR adds short training instructions for using the experiment client and tellina making use of existing authoritative resources.

This was discussed in one of our weekly meetings.

### CODE REVIEW

File: client_side/.infrastructure/infrastructure.sh [hunk 1]
```diff
@@ -200,18 +200,25 @@ check_and_update_training_status() {
 # - The directory that they should be performing tasks on.
 infra_training() {
   echo ${HLINE}
-  echo "For this task, please follow the instructions at"
-  echo "${EXPERIMENT_HOME_URL}/experiment.html"
-  echo "to learn how to do the experiment."
+  echo "This terminal is the interface for the experiment."
+  echo "At any point, run \"helpme\" to see a list of commands available to you."
+  echo ""
+  echo "For each task, we ask you to a corresponding bash one-liner satisfying the prompt, just like you would ordinarily."
```
Comment (anhnamtran) [inline client_side/.infrastructure/infrastructure.sh]: "we ask you to (is something missing here?) a corresponding ..."

File: client_side/.infrastructure/infrastructure.sh [hunk 2]
```diff
@@ -200,18 +200,25 @@ check_and_update_training_status() {
...
+  echo "Please stay in the current test directory already provided, the file system changes between each one-liner is reset after each command."
+  echo "If the one-liner is accepted, the experiment will continue to the next task."
+  echo "If the one-liner does not produce the expected changes, a window with the expected diff will be shown to you so you can retry."
```
Comment (anhnamtran) [inline client_side/.infrastructure/infrastructure.sh]: Maybe "the expected diff" should be "the difference between your output and what the expected output" (or a bit more concise, but verbose still).

**Review by anhnamtran — CHANGES_REQUESTED** [review]

File: client_side/.infrastructure/infrastructure.sh (hunk 1 above)
Comment (Thomsch) [reply client_side/.infrastructure/infrastructure.sh]: Good catch! Thank you 🙏

File: client_side/.infrastructure/infrastructure.sh (hunk 2 above)
Comment (Thomsch) [reply client_side/.infrastructure/infrastructure.sh]: Brilliant! I like your suggestion quite a lot. It's more precise and descriptive. Thanks!

**Review by anhnamtran — APPROVED** [review]
LGTM

**Pull request merged**